### DISCUSSION TITLE
How Can I get a selected CellValue In widgets.Table？

### DISCUSSION THREAD

**Discussion by nemofusion** [fyne-io/fyne#3438] [Q&A] [answered]
How Can I get a selected CellValue In widgets.Table?
widget.NewTable().selectedCell just give me that the row Position And column Position, how can I get that cellValue ?  thanks for help

**Comment by andydotxyz** [comment, marked as answer]
The table does not store your data, so you cannot ask it for value at a position.
The widget renders what is present in a backing data store of some sort - so you should ask your data what is at the position.
The `OnSelected` and `OnUnSelected` help you know when the selection changes, but we cannot tell you what data was there.

**Comment by nemofusion** [comment]
thx, after i send this , i just figure out how to solve this, thx so much!
